web/pgadmin/tools/sqleditor/static/js/components/QueryToolComponent.jsx

```diff
@@ -391,6 +391,13 @@ export default function QueryToolComponent({params, pgWindow, pgAdmin, selectedN
     window.addEventListener('unload', closeConn);
 
     const pushHistory = (h)=>{
+      // Do not store query text if max lenght exceeds.
+      if(h?.query?.length > MAX_QUERY_LENGTH) {
+        h = {
+          ...h,
+          query: gettext('-- Query text not stored as it exceeds maximum length')
```

Comment:
`maximum length of %s`

Reply:
Do you mean mentioning the max length as well?